### PR TITLE
add text & link explaining slow time

### PR DESCRIPTION
**Pull request by rudiejd**
## Motivation

https://github.com/transitmatters/t-performance-dash/issues/850
## Changes

/system/slowzones:
image

/[line]/slowzones
image

## Testing Instructions

Navigate to the system slowzones page and line slowzone pages and ensure that the explainer text is displayed correctly and that the link works

### CODE REVIEW

File: modules/slowzones/SlowZonesDetails.tsx [hunk 1]
```diff
@@ -59,6 +60,13 @@ export function SlowZonesDetails() {
       <ChartPageDiv>
         <WidgetDiv>
           <WidgetTitle title="Total slow time" />
+          <Link
```
Comment (devinmatte) [inline modules/slowzones/SlowZonesDetails.tsx:63]: Can you make this open in a new tab?

File: modules/slowzones/SystemSlowZonesDetails.tsx [hunk 2]
```diff
@@ -63,6 +64,12 @@ export function SystemSlowZonesDetails({ showTitle = false }: SystemSlowZonesDet
       <ChartPageDiv>
         <WidgetDiv>
           <WidgetTitle title="Total slow time" />
+          <Link
```
Comment (devinmatte) [inline modules/slowzones/SystemSlowZonesDetails.tsx:67]: Same as above, new tab please?

**Review by devinmatte — COMMENTED** [review]
One comment, otherwise looks good

File: modules/slowzones/SystemSlowZonesDetails.tsx (hunk 2 above)
Comment (rudiejd) [reply modules/slowzones/SystemSlowZonesDetails.tsx:67]: done, this is now opening in a new tab for me

**Review by devinmatte — APPROVED** [review]
LGTM. I can merge for you if you'd like after fixing the merge conflicts

**Comment by rudiejd** [comment]
thank you @devinmatte ! Yep should be able to merge

**Pull request merged**